Source/Core/DolphinQt/Settings/WiiPane.cpp

```diff
@@ -157,6 +158,7 @@ void WiiPane::CreateMisc()
   m_main_layout->addWidget(misc_settings_group);
   m_pal60_mode_checkbox = new QCheckBox(tr("Use PAL60 Mode (EuRGB60)"));
   m_screensaver_checkbox = new QCheckBox(tr("Enable Screen Saver"));
+  m_wiilink_checkbox = new QCheckBox(tr("Enable WiiConnect24"));
```

Comment:
I think the user-visible label should be "Enable WiiLink" or "Enable WiiConnect24 via WiiLink" - for consistency with what the option actually does and its name in the source code. Otherwise it's a bit misleading.

Reply:
Chose the latter option, fits nice and is more descriptive overall.